backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java

```diff
@@ -385,6 +385,8 @@ private List<Condition> jsonFilterQueryConditions(
         } else if (Operator.AND.equals(f.getOperator())) {
           conditions.add(
               and(jsonFilterQueryConditions(table, column, tableAlias, subAlias, f, searchTerms)));
+        } else if (TRIGRAM_SEARCH.equals(f.getOperator())) {
+          conditions.add(jsonSearchConditions(table, TypeUtils.toStringArray(f.getValues())));
```

Comment:
how does this change the search behaviour ? , i'm missing the before/after test change that showt the behavioural change

Reply:
The only new test included should fail before change. What this does is add filter clause for search that previously was ignored.